src/tests/ftest/util/MpioUtils.py

```diff
@@ -79,13 +87,75 @@ def run_romio(self, basepath, hostlist, romio_test_repo):
             ssh.load_system_host_keys()
             ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
             ssh.connect(hostlist[0])
-            ssh_stdin, ssh_stdout, ssh_stderr = ssh.exec_command(env_variables[0] + \
-                                   " && " + env_variables[1] + " && " + env_variables[2] + \
-                                   " && " + env_variables[3] + " && " + env_variables[4] + \
-                                   " && " + run_cmd)
-            print ssh_stdout.read()
-            print ssh_stderr.read()
+            ssh_stdin, ssh_stdout, ssh_stderr = ssh.exec_command(env_variables[0] \
+                                                + " && " + env_variables[1] + \
+                                                " && " + env_variables[2] + \
+                                                " && " + env_variables[3] + \
+                                                " && " + env_variables[4] + \
+                                                " && " + run_cmd)
+            print (ssh_stdout.read())
+            print (ssh_stderr.read())
+
+        except (IOError, OSError, paramiko.SSHException, socket.error) as error:
+            raise MpioFailed("<ROMIO Test FAILED> \nException occurred: {}"\
+                             .format(str(error)))
+
+    def run_llnl_mpi4py(self, basepath, hostfile, pool_uuid, test_repo,
+                        test_name):
+        """
+            Running LLNL and MPI4PY testsuites
+            Function Arguments:
+                basepath          --path where all daos and it's dependencies
+                                    can be fetched from
+                hostfile          --client hostfile
+                pool_uuid         --Pool UUID
+                test_repo         --test repo location
+                display_output    --print output.
```

Comment:
Information missing for test_name.
No display_output argument any more

Reply:
display_output argument is removed bcoz we need to display output in order to parse through the output message and look for error messages. So it won't make sense to disable "display_output" at any moment.
Will add test_name definition.

Reply:
That is correct so we need to remove that display_output information from doc string as well.

Reply:
Yes, I saw it. Will do.